plugins/com.google.cloud.tools.eclipse.appengine.libraries.ui/src/com/google/cloud/tools/eclipse/appengine/libraries/ui/LibrarySelectorGroup.java

```diff
@@ -70,7 +70,7 @@ public LibrarySelectorGroup(Composite parentContainer, String groupName) {
     Collection<Library> availableLibraries = CloudLibraries.getLibraries(groupName);
     this.availableLibraries = new LinkedHashMap<>();
     for (Library library : availableLibraries) {
-      if (!restrictedEnvironment || !library.getJavaVersion().equals("1.8")) {
+      if (!restrictedEnvironment || "http".equals(library.getTransport())) {
```

Comment:
I have to admit to finding this encoded knowledge a bit hard to remember. Perhaps the javadocs for `restrictedEnvironment` should be updated.  Or maybe we should pass in an instance of `AppEngineRuntime` — or create a `Runtime` enum for the different platforms.

Reply:
need to refactor a bit, but meanwhile updated javadoc